staging/src/k8s.io/client-go/metadata/metadatainformer/informer.go

```diff
@@ -58,12 +43,95 @@ type metadataSharedInformerFactory struct {
 	informers map[schema.GroupVersionResource]informers.GenericInformer
 	// startedInformers is used for tracking which informers have been started.
 	// This allows Start() to be called multiple times safely.
-	startedInformers map[schema.GroupVersionResource]bool
-	tweakListOptions TweakListOptionsFunc
+	startedInformers   map[schema.GroupVersionResource]bool
+	stoppableInformers map[schema.GroupVersionResource]cache.DoneChannel
+	tweakListOptions   TweakListOptionsFunc
+	stopOnError        cache.StopOnErrorFunc
 }
 
 var _ SharedInformerFactory = &metadataSharedInformerFactory{}
 
+// WithStopOnError sets the stopOnListErrFunc that is added to the stop options
```

Comment:
```suggestion
// WithStopOnError sets the stopOnErrorFunc that is added to the stop options
```
A typo?

Reply:
Yep. Fixed here and in the dynamicinformer, thanks!